src/plugins/requestAuthPlugin.js

```diff
@@ -47,22 +53,20 @@ export const requestAuthorizer = auth$ => request => {
 	// always need oauth_token, even if it's an anonymous (pre-reg) token
 	// This is 'deferred' because we don't want to start fetching the token
 	// before we know that it's needed
-	const request$ = Rx.Observable.of(request);
-	const authType = request.state.oauth_token && 'cookie';
-
 	request.log(['info', 'auth'], 'Checking for oauth_token in request');
-	return Rx.Observable.if(
-		() => authType,
-		request$
-			.do(() => request.log(['info', 'auth'], `Request contains auth token (${authType})`)),
-		request$
-			.do(() => request.log(['info', 'auth'], 'Request does not contain auth token'))
-			.flatMap(request =>
-				auth$(request)
-					.do(applyAuthState(request, request.plugins.requestAuth.reply))
-					.map(() => request)
-			)
-	);
+
+	const authType = request.state.oauth_token && 'oauth_token' ||
+		request.state.MEETUP_MEMBER && 'MEETUP_MEMBER';
```

Comment:
at this point in the request cycle, `request.state.MEETUP_MEMBER` will contain the right value. Do you think I should just add an `isDev` check here instead of manipulating `request.state.MEETUP_MEMBER`?